### PR TITLE
Improve documentations for tolerate-non-existent-backend-{service,action}

### PR DESCRIPTION
**Pull request by guessi**
### Issue

#3440

### Description

See #3440 for more details.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [X] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [X] Manually tested
- [X] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by M00nF1sh — APPROVED** [review]
/lgtm
/approve

**Comment by guessi** [comment]
rebased and revised commit message, trying to get `do-not-merge/invalid-commit-message ` passed.

**Comment by guessi** [comment]
@oliviassss @M00nF1sh

I've rebase the commit, and tried to shorten my commit message to get `do-not-merge/invalid-commit-message` check pass, but it doesn't looks like it could make bot happy. Let me know what should I do next, thanks!

**Comment by guessi** [comment]
Umm... looks like it violate [this rule](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#do-not-use-github-keywords-or-mentions-within-your-commit-message), let me try again.

**Comment by guessi** [comment]
Nice, `do-not-merge/invalid-commit-message` [removed](https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3442?notification_referrer_id=NT_kwDOAA8FX7E4MDk3NDAyODQ1Ojk4NDQxNQ#event-10997969558), but `/lgtm` removed as well... 😶‍🌫️

**Review by M00nF1sh — APPROVED** [review]
/lgtm
/approve
thanks for making this contribution
